templates/platforms/android-studio/app/src/main/AndroidManifest.xml.hbs

```diff
@@ -12,6 +12,7 @@
         android:theme="@style/AppTheme">
         <activity
             android:configChanges="orientation|keyboardHidden"
+            android:exported="true"
```

Comment:
The rest of this looks straightforward, though I'm curious what this does?

Reply:
It's something we have to set for Android 12 and higher. It should be false now that I look back at it tho:
https://developer.android.com/guide/topics/manifest/service-element